base/interactiveutil.jl

```diff
@@ -64,7 +64,27 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-edit(f)          = edit(functionloc(f)...)
+function edit(f)
+    ms = methods(f)
```

Comment:
The original implementation worked on more than just Function. Maybe this implementation should be just for `edit(f::Function)`?